docker/Dockerfile.communication

```diff
@@ -0,0 +1,18 @@
+FROM python:3.8-slim
+
+ARG REACT_APP_COMMUNICATIONS_BASE_URI=
+ARG TERRY_BRANCH=master
+EXPOSE 80
+
+# Setup terry
+ADD setup.sh /
+RUN /setup.sh communication
+
+ADD start.sh /
+ADD nginx.conf /etc/nginx/nginx.conf
+ADD default.config.yaml /
+
+# Communication backend executable
+ADD terry-communication-backend /
```

Comment:
This file cannot be merged with the other `Dockerfile` since Docker doesn't support conditional file adding.